.github/workflows/goreleaser-ci.yml

```diff
@@ -0,0 +1,35 @@
+
+name: GoReleaser CI
+
+on:
+  push:
+    branches: 
+      - main
+  pull_request:
+    branches:
+      - '**'
+  workflow_dispatch:
+
+jobs:
+  goreleaser_ci_build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [1.17.1]
+
+    steps:
+      - 
+        name: Install go
```

Comment:
Please match the style of boulder-ci.yaml (or update it to match this style; I don't care which wins as long as we're consistent).

Reply:
Changed the style to match boulder-ci.yaml